bin/s2nc.c

```diff
@@ -238,6 +245,31 @@ int main(int argc, char *const *argv)
 
     struct s2n_connection *conn = s2n_connection_new(S2N_CLIENT);
 
+    if (mfl_value > 0) {
+        switch(mfl_value) {
+            case 512:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_512;
+                break;
+            case 1024:
+                mfl_code = S2N_TLS_MAX_FRAG_LEN_1024;
```

Comment:
Actually, after some thought. 
Since this is for thes2n cli...  (It would be great if we have more data on this) I assume if a user is using cli not directly implementing s2n using APIs, cli should not require users to go to source code and check APIs in s2n.h file. 

The user should be able to complete what they want to do just through cli's -h usage comments.